exasol_integration_test_docker_environment/lib/test_environment/create_certificates/create_ssl_certificates_task.py

```diff
@@ -56,7 +56,7 @@ def run_task(self):
 
         self.return_object(self.volume_info)
 
-    def build_image(self) -> Generator["BaseTask", PickleTarget, Set[str]]:
+    def build_image(self) -> Generator[BaseTask, None, Set[str]]:
```

Comment:
Return Type and send Type of the Generator Type don't makes sense, the send Type might be an artifact of how Luigi or our framework around it works, but the return type should be probably Set[Imageainfo]

Reply:
Changed to
```
Generator[BaseTask, None, Set[Imageainfo]]:
```